casanovo/data/db_utils.py

```
@@ -76,6 +76,21 @@ def __init__(
         self.fixed_mods, self.var_mods, self.swap_map = _construct_mods_dict(
             allowed_fixed_mods, allowed_var_mods
         )
+        # Warn about modified residues in the model vocabulary
+        configured_mods = set()
+        for mods_str in (allowed_fixed_mods, allowed_var_mods):
+            for mod in mods_str.split(","):
+                parts = mod.split(":")
+                if len(parts) == 2:
+                    configured_mods.add(parts[1])
+        for residue in tokenizer.residues.keys():
```

Ruff (0.15.6)

[warning] Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)